### ISSUE TITLE
VLE Unit Plan UI bugs

### ISSUE THREAD

**Issue by breity** [WISE-Community/WISE#2836]
- The Notebook FAB button and Report toggle scroll with the page on VLE Unit Plan. They should remain at the bottom of the window.

![image](https://user-images.githubusercontent.com/297450/101706124-521d4c80-3a3d-11eb-8253-e7c57027252a.png)

- Lesson title text hover state (bottom border) doesn't work well for multiline titles.

![image](https://user-images.githubusercontent.com/297450/101706204-77aa5600-3a3d-11eb-9121-8c8b126de1e9.png)

- Enter key doesn't toggle lessons opened/closed (not keyboard accessible).
